tests/test_build_html.py

```diff
@@ -24,10 +24,7 @@
 HTML_PARSER = HTMLParser(TREE_BUILDER, namespaceHTMLElements=False)
 
 ENV_WARNINGS = """\
-(%(root)s/autodoc_fodder.py:docstring of autodoc_fodder.MarkupError:\\d+: \
-WARNING: duplicate object description of autodoc_fodder.MarkupError, other \
-instance in %(root)s/autodoc.rst, use :noindex: for one of them
```

Comment:
I think this warning line will never appear because autodoc.rst is not exist in the `roots/test-warnings`.